sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/LogRecordExporterConfiguration.java

```diff
@@ -27,6 +27,7 @@ final class LogRecordExporterConfiguration {
 
   static {
     EXPORTER_ARTIFACT_ID_BY_NAME = new HashMap<>();
+    EXPORTER_ARTIFACT_ID_BY_NAME.put("console", "opentelemetry-exporter-logging");
```

Comment:
I'd be a little hesitant in renaming artifacts. It's definitely caused a lot of confusion in the past when suddenly an artifact no longer exists for a version, and it's been renamed. This has often led to people mixing and matching artifact versions, which is a significant foot gun for users.

Reply:
An argument against renaming artifacts is that it will cause the class names to be misaligned: 
- Artifact named `opentelemetry-exporter-console`
- Classes named `Logging{Signal}Exporter`

Of course we could introduce new classes named `Console{Signal}Exporter` and deprecate the `Logging{Signal}Exporter` variations.

Reply:
I find the name `opentelemetry-exporter-logging` to be regularly confusing (I keep thinking it's the log exporter).

> I'd be a little hesitant in renaming artifacts. It's definitely caused a lot of confusion in the past when suddenly an artifact no longer exists for a version, and it's been renamed. This has often led to people mixing and matching artifact versions, which is a significant foot gun for users.

this is a great point though

maybe we can have both `opentelemetry-exporter-console` and keep publishing (a deprecated) `opentelemetry-exporter-logging` artifact (which can have a dependency on `opentelemetry-exporter-console` if that's helpful for reducing duplication)?

Reply:
> maybe we can have both opentelemetry-exporter-console and keep publishing (a deprecated) opentelemetry-exporter-logging artifact (which can have a dependency on opentelemetry-exporter-console if that's helpful for reducing duplication)?

That's an interesting idea.. 

While we're on the subject of making the logging exporters less confusing / more useful, would probably be good to optionally give the user more control over where the logs go. Could have an optional `Consumer<String> logHandler` argument which defaults to sending to JUL, but could also be configured to go to the user's desired logger (SLF4J, System.out, etc). Could apply the same principle to the logging OTLP exporter.

Reply:
Opened #6263 to discuss further and track. Shouldn't be a block for this PR tho.